tests/testqPrefGeneral.cpp

```diff
@@ -24,7 +24,7 @@ void TestQPrefGeneral::test_struct_get()
 	auto tst = qPrefGeneral::instance();
 
 	prefs.auto_recalculate_thumbnails = true;
-	prefs.default_cylinder = copy_qstring("new base11");
+//	prefs.default_cylinder = copy_qstring("new base11");
```

Comment:
here and everywhere else - simply commenting out a line like this may make sense during development, and sometimes may make sense in a commit, but then there should be a comment alongside that tells us _why_ this was commented out...